commit 51880df577d0645219007c3f84075012361bfa25 (refs/changes/18/5018/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-10-31T09:51:53-07:00 (11 months ago)
    
    TRITON-920 triton repos manifest maint Halloween 2018 (new forks for triton prometheus core service image)

diff --git a/repos.json b/repos.json
index 3aa6958..9f1e391 100644
--- a/repos.json
+++ b/repos.json
@@ -77,6 +77,9 @@
                 "meta": true
             }
         },
+        {
+            "name": "fsnotify"
+        },
         {
             "name": "java-triton"
         },
@@ -379,6 +382,9 @@
         {
             "name": "postgres"
         },
+        {
+            "name": "prometheus"
+        },
         {
             "name": "registrar",
             "labels": {
